src/content/release-notes.mdx

```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.27.0
+
+9 December 2024
+
+This version is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Remove code for our deprecated Quickstarts feature <!-- 8842 -->
```

Comment:
There is another breaking change about to overwrite the container images in the helm chart. I would link to the airgap guide saying that's the new way to overwrite images

Reply:
I think including this breaking change is still pending